test/unit/auth/token-verifier.spec.ts

```diff
@@ -46,13 +47,18 @@ const ONE_HOUR_IN_SECONDS = 60 * 60;
 /**
  * Returns a mocked out success response from the URL containing the public keys for the Google certs.
  *
+ * @param {string=} path URL path to which the mock request should be made. If not specified, defaults
+ *   to the URL path of ID token public key certificates.
  * @return {Object} A nock response object.
  */
-function mockFetchPublicKeys(): nock.Scope {
+function mockFetchPublicKeys(path?: string): nock.Scope {
```

Comment:
You can directly set a default value from here.

Reply:
Done